.github/workflows/test.yml

```diff
@@ -27,20 +27,9 @@ jobs:
     - name: install dependencies
       run: pip install numpy
 
-    - name: build splinepy linux - gcc10
-      if: matrix.os == 'ubuntu-20.04'
+    - name: build splinepy minimal/debug/warning
       run: |
-        CC=gcc-10 CXX=g++-10 python3 setup.py install --minimal --debug --enable_warning
-
-    - name: build splinepy macos
-      if: matrix.os == 'macos-latest'
-      run: |
-        python3 setup.py develop --minimal --debug --enable_warning
-
-    - name: build splinepy windows
-      if: matrix.os == 'windows-latest'
-      run: |
-        python3 setup.py develop --minimal --enable_warning
+        pip install . -v --config-settings=cmake.args="-DSPLINEPY_MORE=OFF;-DSPLINEPY_ENABLE_WARNINGS=ON" --config-settings=cmake.build-type="Debug"
```

Comment:
So the `--minimal` option is now `-BSPLINEPY_MORE=OFF`.

Should we maybe start to invert the functionality that the option is by default `OFF` and can be set `ON` if the larger build is necessary? 
How useful is it for a normal developer/user to have the full build and not only the minimal?

Or has the problem with the incredibly high RAM and CPU usage for the full build been mitigated?

Reply:
> So the `--minimal` option is now `-BSPLINEPY_MORE=OFF`.
> 
> Should we maybe start to invert the functionality that the option is by default `OFF` and can be set `ON` if the larger build is necessary? How useful is it for a normal developer/user to have the full build and not only the minimal?

I thought about that, but I wanted to keep the  same as distribution build. I am playing with the idea to add a `scripts` or `tools` directory to gather some commands. This modern approach is log

> 
> Or has the problem with the incredibly high RAM and CPU usage for the full build been mitigated?

No. Definitely less, as explicit build of bsplines has significantly reduced. But I think nothing changed for Beziers.